csv.inc.php

```diff
@@ -56,6 +56,22 @@ protected function csvToArray($csv = 'stock.csv')// Access and extract CSV
         return $arrOut;
     }
 // end of csvToArray
+    protected function columnLength($csv = 'stock.csv')
```

Comment:
This currently has no actual output, this should either modify a private variable, or return something.